src/app/dataset-view/dataset-view-menu/dataset-view-menu.component.ts

```diff
@@ -167,7 +178,18 @@ export class DatasetViewMenuComponent implements OnInit, AfterViewInit {
     }
 
     private initClipboardHints(): void {
-        this.clipboardKamuCli = `kamu pull kamu.dev/${this.datasetBasics.alias}`;
-        this.clipboardKafka = `https://api.kamu.dev/kafka/${this.datasetBasics.alias}`;
+        this.clipboardReference = `https://kamu.dev/${this.datasetBasics.alias}`;
```

Comment:
We should un-hardcode `kamu.dev` - frontend host should come from browser URL
(as a temporary solution until correct URLs are provided via API)

Reply:
Changed  to

> this.clipboardReference = `https://${location.host}/${this.datasetBasics.alias}`;